backend/public-graph/graph/resolver.go

```diff
@@ -2900,7 +2900,7 @@ func (r *Resolver) ProcessPayload(ctx context.Context, sessionSecureID string, e
 	}
 
 	// if the session changed from excluded to not excluded, it is viewable, so send any relevant alerts
-	if sessionObj.Excluded && !excluded {
+	if !excluded && (sessionObj.Excluded || (sessionObj.FirstTime != nil && *sessionObj.FirstTime)) {
```

Comment:
@mayberryzane noticed these changes were added to avoid spending too much CPU in `SendSessionInitAlert`. I think this would fire every payload where the session is a user's first session, but still excludes other following sesssions

Reply:
looks good, you're just including first time sessions to this branch